tests/backend_test.py

```diff
@@ -988,46 +988,6 @@ def lift_perm(p: dict[int, int]) -> np.ndarray:
     return pm
 
 
-@pytest.mark.skipif(skip_remote_tests, reason=REASON)
-def test_compilation_correctness(brisbane_backend: IBMQBackend) -> None:
```

Comment:
LightSabre Pass does not return any information regarding the implicit qubit permutation. If a circuit has `Measure` operations, as we'd expect for hardware, then this is not an issue, as all post-processing is completed based on output bits from `Measures`, which are still measuring the same states.

This test confirmed that the compilation, including SWAP operations, did not change the unitary, action, up to permutations, for a backend with some architecture.  In practice we do not expect users to want to do unitary simulation with qubit swapping, so I think we can reasonably remove this test.

Reply:
Is this potentially an issues for other cases? Should this be documented somewhere?

Reply:
I've added this test back in, but edited it to only check the routing is correct